doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+
++--------------------------------+----------------------------------------------------------+
+|Component                       |  Build target                                            |
++================================+==========================================================+
+| nRF5340 SoC - Application core |``thingy53_nrf5340_cpuapp`` for the secure domain         |
+|                                |                                                          |
+|                                |``thingy53_nrf5340_cpuapp_ns`` for the non-secure version |
++--------------------------------+----------------------------------------------------------+
+| nRF5340 SoC - Network core     |``thingy91_nrf52840``                                     |
++--------------------------------+----------------------------------------------------------+
+
+The table below shows the different types of build files that are generated and the different scenarios in which they are used:
+
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| File                        | File format                          | Programming scenario                                               |
++=============================+======================================+====================================================================+
+| :file:`merged.hex`          | Full image, HEX format               | Using an external debug probe and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_signed.hex`      | MCUboot compatible image, HEX format | Using the built-in bootloader and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_update.bin`      | MCUboot compatible image, HEX format | Using the built-in bootloader and the mcumgr command line tool     |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`dfu_application.zip` | MCUboot compatible image, binary     | Using nRF Programmer for Android and iOS or nRF Connect Programmer |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+
+There are multiple methods of programming a sample or application onto a Thingy:53.
+You can choose the method based on the availability or absence of an external debug probe to program.
+
+.. note::
+
+   If you do not have an external debug probe available to program the Thingy:53, you can directly program the device over USB using the built-in MCUboot bootloader and nRF Connect Programmer.
```

Comment:
I would add a note here that this will work for thingy53 compatible samples (do we have list of these somewhere?)

Reply:
Agreed.  Suggest changing it to:

 If you do not have an external debug probe available to program the Thingy:53, you can directly program Thingy:53 compatible samples to the device over USB using the built-in MCUboot bootloader and nRF Connect Programmer.
 
 Maybe the following list should be added somewhere in the Working with Thingy:53 section. 

-  nRF Machine Learning
- Thingy:53: Matter weather station
- Bluetooth: Peripheral LBS
- Bluetooth: Peripheral UART
- Bluetooth: Mesh light
- Bluetooth: Mesh light switch

Reply:
Added these directly to the note under "Some compatible samples"